src/libertem/analysis/sumsig.py

```diff
@@ -1,9 +1,36 @@
 from libertem.viz import visualize_simple
 from .base import BaseAnalysis, AnalysisResult, AnalysisResultSet
 import libertem.udf.sumsigudf as sumsigudf
+from .helper import GeneratorHelper
 
 
-class SumSigAnalysis(BaseAnalysis):
+class SumSigTemplate(GeneratorHelper):
+
+    short_name = "sumsig"
+
+    def __init__(self, params):
+        self.params = params
+
+    def get_dependency(self):
+        return "from libertem.analysis import SumSigAnalysis"
+
+    def get_docs(self):
+        docs = ["# SumSig Analysis",
+                "***about sumsig analysis ***"]
```

Comment:
If you are adding helpful descriptions for each analysis, maybe it would be good to also display those in the GUI, maybe available as an "about analysis" button as very short online help? That would be very nice!

Reply:
I was trying to get docs as @uellue suggested, from `Context`. I will add descriptions for analysis which doesn't have right now.

Reply:
That's good! I mean where the description comes from doesn't really matter - what I'm saying is that it would be very cool to **also** display it in the GUI, if we already have it available for generating the notebooks